src/utils/mod.rs

```diff
@@ -36,10 +36,9 @@ pub(crate) const APP_USER_AGENT: &str = concat!(
 pub(crate) fn report_error(err: &anyhow::Error) {
     if std::env::var("SENTRY_DSN").is_ok() {
         sentry_anyhow::capture_anyhow(err);
-    } else {
-        // Debug-format for anyhow errors includes context & backtrace
-        error!("{:?}", err);
     }
+    // Debug-format for anyhow errors includes context & backtrace
+    error!("{:?}", err);
```

Comment:
this will lead to the error being reported twice to sentry, since it's also hooked into `tracing`